recipes/xgcm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xgcm
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xgcm/xgcm-{{ version }}.tar.gz
+  sha256: 1eb5e716530b63fbe7da129fd20fa85451c527f61230df172e419eb100ce2f63
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
```

Comment:
This is probably not needed but it is listed as `setup_requires`, we need to check with upstream.